packages/dcos-log/build

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+export GOPATH=$GOPATH:/pkg
+mkdir -p /pkg/src/github.com/dcos
+# Create the GOPATH for the go tool to work properly
+mv /pkg/src/dcos-log /pkg/src/github.com/dcos/
+cd /pkg/src/github.com/dcos/dcos-log
+
+make install
+# Copy the build from the bin to the correct place
+cp -r /pkg/bin/ "$PKG_PATH"
+
+master_service=${PKG_PATH}/dcos.target.wants_master/dcos-log.service
+slave_service=${PKG_PATH}/dcos.target.wants_slave/dcos-log.service
+slave_public_service=${PKG_PATH}/dcos.target.wants_slave_public/dcos-log.service
+
+master_socket_service=${PKG_PATH}/dcos.target.wants_master/dcos-log.socket
+slave_socket_service=${PKG_PATH}/dcos.target.wants_slave/dcos-log.socket
+slave_public_socket_service=${PKG_PATH}/dcos.target.wants_slave_public/dcos-log.socket
+
+mkdir -p $(dirname $master_service)
+mkdir -p $(dirname $slave_service)
+mkdir -p $(dirname $slave_public_service)
+
+cat <<EOF | tee $master_service
+[Unit]
+Description=DC/OS Log service: DC/OS logging service
+[Service]
+EnvironmentFile=/opt/mesosphere/environment
+Restart=always
+StartLimitInterval=0
+RestartSec=5
+User=dcos_log
+ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-log
```

Comment:
unfortunately we need 2 different system accounts for running the same service on master and agent nodes. The bootstrap does not know about roles at all, and we need to communicate the node role somehow. The best way to do this is to run bootstrap script with different parameters, this is how we bootstrap service account now.
cc @alberts